Source/MarkdownGenerator/src/main/java/net/steppschuh/markdowngenerator/list/OrderedListItem.java

```diff
@@ -0,0 +1,20 @@
+package net.steppschuh.markdowngenerator.list;
+
+import net.steppschuh.markdowngenerator.list.UnorderedListItem;
+
+public class OrderedListItem extends UnorderedListItem {
+
+    private int index = 1;
+
+    public OrderedListItem(Object value) {
```

Comment:
Add the index to the constructor, I think there's no valid use case for having a list item without an explicitly set index. You'll probably need this constructor when implementing `OrderedList.serialize`